utils.js

```diff
@@ -708,7 +716,7 @@ function formatThread(data) {
     muteSettings: data.muteSettings,
     isCanonicalUser: data.is_canonical_user,
     isCanonical: data.is_canonical,
-    canonicalFbid: data.canonical_fbid,
+    canonicalFbid: formatID(data.canonical_fbid),
```

Comment:
Found a way around it using a method similar to the one used on `snippetSender`. Although this means that applications which previously checked whether `canonicalFbid` was undefined would not work as it will now just be an empty string (IMO this is better but may break someones app).